igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerMonitor.groovy

```diff
@@ -101,7 +101,11 @@ class DockerMonitor extends CommonPollingMonitor<ImageDelta, DockerPollingDelta>
         Set<String> cachedImages = cache.getImages(account)
 
         long startTime = System.currentTimeMillis()
-        List<TaggedImage> images = AuthenticatedRequest.allowAnonymous { dockerRegistryAccounts.service.getImagesByAccount(account) }
+        List<TaggedImage> images = AuthenticatedRequest.allowAnonymous { dockerRegistryAccounts.service.getImagesByAccount(account, true) }
```

Comment:
Let's do this in two separate changes - first, roll out the timing. Then, change the API call. That way we get a good indication of if the call takes longer

Reply:
done!